src/app/Components/ArtworkGrids/ArtworkSocialSignal.tsx

```diff
@@ -21,31 +21,19 @@ export const ArtworkSocialSignal: React.FC<ArtworkSocialSignalProps> = ({
   switch (true) {
     case curatorsPick && !hideCuratorsPick:
       return (
-        <Box
-          px={0.5}
-          pb="2px"
-          alignSelf="flex-start"
-          borderRadius={3}
-          borderWidth={1}
-          borderColor={primaryColor}
-        >
-          <Text color={primaryColor} variant="xxs">
+        <Box alignItems="center" flexDirection="row">
+          <FireIcon />
+          <Text color={primaryColor} variant="xs" pl="3px" textAlign="center">
             Curators’ Pick
           </Text>
         </Box>
       )
 
     case increasedInterest && !hideIncreasedInterest:
       return (
-        <Box
-          px={0.5}
-          pb="2px"
-          alignSelf="flex-start"
-          borderRadius={3}
-          borderWidth={1}
-          borderColor={primaryColor}
-        >
-          <Text color={primaryColor} variant="xxs">
+        <Box flexDirection="row" alignItems="center">
+          <ArrowUpRightIcon />
+          <Text color={primaryColor} variant="xs" pl="3px">
```

Comment:
Would this work as well? IMO it's better to use an empty space character instead of padding. But I don't feel very strong about it.

```suggestion
          <Text color={primaryColor} variant="xs">{" "}Increased Interest</Text>
```